packages/cli/src/lib/config.ts

```diff
@@ -29,25 +31,41 @@ const LEGACY_CONFIG_FOLDER = join(homedir(), '.llama-pack');
 const LEGACY_CONFIG_NAME = 'llama-pack-config.json';
 const LEGACY_CONFIG_FILE_PATH = join(LEGACY_CONFIG_FOLDER, LEGACY_CONFIG_NAME);
 const DEFAULT_JDK_FOLDER = join(DEFAULT_CONFIG_FOLDER, 'jdk');
+const DEFAULT_SDK_FOLDER = join(DEFAULT_CONFIG_FOLDER, 'android_sdk');
 
 async function createConfig(prompt: Prompt = new InquirerPrompt()): Promise<Config> {
-  const jdkInstallRequest = await prompt.promptConfirm('Do you want Bubblewrap to install JDK? ' +
-    '(Enter "No" to use your JDK installation)', true);
+  const jdkInstallRequest = await prompt.promptConfirm(messages.promptInstallJdk, true);
 
   let jdkPath;
   if (!jdkInstallRequest) {
-    jdkPath = await prompt.promptInput('Path to your existing JDK:', null,
+    jdkPath = await prompt.promptInput(messages.promptJdkPath, null,
         JdkHelper.validatePath);
   } else {
-    await fsPromises.mkdir(DEFAULT_JDK_FOLDER);
-    console.log(`Downloading JDK 8 to ${DEFAULT_JDK_FOLDER}`);
+    await fsPromises.mkdir(DEFAULT_JDK_FOLDER, {recursive: true});
```

Comment:
You may get a merge conflict with https://github.com/GoogleChromeLabs/bubblewrap/pull/375/files on this line (Andre added `{recursive: true}` too).